src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMailboxes.scala

```diff
@@ -21,6 +21,14 @@ object JmapMailboxes {
     JmapAuthentication.authenticatedQuery("getMailboxes", "/jmap")
       .body(StringBody("""[["getMailboxes", {}, "#0"]]"""))
 
+  def getMailboxIdByName(name: Name) =
+    JmapAuthentication.authenticatedQuery("getMailboxes", "/jmap")
+      .body(StringBody("""[["getMailboxes", {}, "#0"]]"""))
+      .check(getMailboxIdCheck(name))
+
+  def getMailboxIdCheck(name: Name) =
```

Comment:
it's not a getter, it extracts and saves

Reply:
I don't understand this function on what json does jsonPath work on ?